src/App.js

```diff
@@ -1,26 +1,12 @@
-import logo from './logo.svg';
-import './App.css';
+import './index.css';
+import React from 'react';
+import Design from './components/Calculator';
 
 function App() {
```

Comment:
Please be sure to make the necessary changes in your  `App.js` file component so that App() function is class-based 🤓